src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3863,7 +3868,8 @@ def main():
         cgroup = CgroupUtils(hostname, vnode)
         pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Cgroup utility class instantiated" %
                    (caller_name()))
-        pbs.logmsg(pbs.EVENT_DEBUG4, "%s: %s" % (caller_name(), repr(cgroup)))
+        # Avoid printing repr by default due to its size
+        # pbs.logmsg(pbs.EVENT_DEBUG4, "%s: %s" % (caller_name(), repr(cgroup)))
```

Comment:
I'll make this comment and leave it up to you if you want to do anything with it.  If this was main PBS C code, and debugging code was commented out, it would have to be removed before check-in.  Since this hook is considered PBS supported, should we do the same here?

Reply:
I see your point, but the problem with removing the code is that someone might need it again in the future when they are debugging a problem. Of course, they could always look back at previous versions to find what they need. I debated (with myself) over whether to remove it or comment it out. If it's our policy to remove such code in C, then it should be the case for all code. I'll remove the comments.